files/galaxy/tpv/destinations.yml.j2

```diff
@@ -268,7 +268,7 @@ destinations:
 
   pulsar_cz02_tpv:
     inherits: pulsar_default
-    runner: pulsar_eu_cz02
+    runner: pulsar_eu_cz01
```

Comment:
but the destination is still called `02`?

Reply:
2 destinations, and both of them are using the same runner. One destination has a GPU, and one does not.

Reply:
Exactly as Sanjay said. Should be fine, right? At least I didn't find another way how to squize it into one destination, though I don't need 2 runners nor 2 managers in pulsar.

Reply:
Are there two separate PBS queues?

One can use TPV `rules` and `conditionals` to configure/overwrite things, but this might be messy and cause unwanted issues if not thoroughly tested. Having two exclusive destinations is fine and maintainable.

Reply:
Well, yeah, I guess it should be possible, but it's a bit overkill for this case. Maybe one day, when I master TPV :)